src/lib/comms/sol-network-util.h

```diff
@@ -55,10 +55,10 @@ sol_bluetooth_is_addr_str(const char *str)
     const char *p = str;
 
     while (*str) {
-        if (!isxdigit(*str++))
+        if (!isxdigit((int)*str++))
             return false;
 
-        if (!isxdigit(*str++))
+        if (!isxdigit((int)*str++))
```

Comment:
On Tue, 10 May 2016 07:46:50 -0700, Vinicius Gomes wrote:
> >              return false;
> >  
> > -        if (!isxdigit(*str++))
> > +        if (!isxdigit((int)*str++))
> 
> I personally prefer this as it is, but no strong feelings about it.

These ctype macros tend to complain some times when the parameter passed
is a plain 'char'. At the very least, make sure to cast it to an
unsigned one.